src/parser/rules/mod.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::borrowed_box)]
```

Comment:
You can also change the borrowed box to `&dyn WhateverTheTraitIs` and then change the caller to `&**var_name`